examples/kms/vault/kms-config.yaml

```diff
@@ -9,6 +9,7 @@ data:
         "vaultAddress": "http://vault.default.svc.cluster.local:8200",
         "vaultAuthPath": "/v1/auth/kubernetes/login",
         "vaultRole": "csi-kubernetes",
+        "vaultToken": "sample_root_token_id",
```

Comment:
The change here as an example confuses me (at least), as it does not remove the auth path and role, and adds a different auth mechanism (token) into the mix. If we decide to fix order of authentication methods as per the other comment in `InitVaultKMS` then to test tokens or authPath we would need another example kms-config YAML that holds these mutually exclusive.

Also, as it stands, if post this PR we are only testing token based authentication (which is what it looks like on a brief inspection of the code), then from an E2E standpoint we are not testing the user facing code in  `getAccessToken`. I would prefer if we tested user facing options in E2E rather than developer options, if the choice is one or the other.